packages/components/src/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -225,3 +254,42 @@ export const ConfirmationModal = forwardRef(function ConfirmationModalInternal(
     </Modal>
   );
 });
+
+function createKeyDownHandler(
+  option:
+    | Array<
+        KeyboardEventCompareratorOption | KeyboardEventCompareratorOption["key"]
+      >
+    | KeyboardEventCompareratorOption
+    | KeyboardEventCompareratorOption["key"],
```

Comment:
How would you feel about something along the lines of

```suggestion
  keys: KeyComparerator[] | KeyComparerator
```